### DISCUSSION TITLE
Clarification on SERVER_URL in `secrets.properties`

### DISCUSSION THREAD

**Discussion by chesh-ire** [Vishwajith-Shettigar/NextGen#47] [Q&A] [answered]
Hi there,
I'm working on setting up the secrets.properties file as per the instructions. I understand that I need to add my `MAPS_API_KEY`, but I'm unsure about what to put for the `SERVER_URL`. Could you please clarify what the expected value for the `SERVER_URL` is? Specifically:

-What kind of URL should this be?
-Should it point to a specific server or service, like for video calls?

**Comment by Vishwajith-Shettigar** [comment, marked as answer]
Hi, thanks for bringing this up.

If you are not working on the video call feature, please add any random characters as a workaround, e.g., `SERVER_URL=pbdwkw`.

Just for your information, `SERVER_URL` holds the URL of the video call server, you can find the repository [here](https://github.com/Vishwajith-Shettigar/video-call-server-node.js).
